src/azure-cli/azure/cli/command_modules/vm/_validators.py

```diff
@@ -706,12 +706,68 @@ def _validate_vm_vmss_accelerated_networking(cli_ctx, namespace):
         size = getattr(namespace, 'size', None) or getattr(namespace, 'vm_sku', None)
         size = size.lower()
 
+        # Deprecated
```

Comment:
it would be nice to add detail why deprecate

Reply:
Let me delete these 3 lines.